src/hotspot/share/prims/jvmtiEnvBase.cpp

```diff
@@ -214,6 +214,7 @@ JvmtiEnvBase::JvmtiEnvBase(jint version) : _env_event_enable() {
 
   // all callbacks initially null
   memset(&_event_callbacks,0,sizeof(jvmtiEventCallbacks));
+  memset(&_ext_event_callbacks, 0, sizeof(jvmtiExtEventCallbacks));
```

Comment:
Looks good.

While you are here could you adjust the existing memset line and add spaces after the commas please. Thanks.

Reply:
Done